modules/virt-how-fs-overhead-affects-space-vm-disks.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/<INSERT ASSEMBLY NAME>.adoc
+
+[id="virt-how-fs-overhead-affects-space-vm-disks_{context}"]
+= How file system overhead affects space for VM disks
+
+When you add a virtual machine disk to a persistent volume (PV) that uses the `Filesystem` volume mode, you must ensure that there is enough space on the PV for:
```

Comment:
should say it's percentage of the PersistentVolumeClaim, sorry about that (the PV might be bigger than the PVC)